flex-backend/api/http/server.go

```diff
@@ -15,50 +15,76 @@ type Server struct {
 }
 
 func (s Server) BuildEndpoints() {
-	http.HandleFunc("/getMovies", handleGetMovies)
-	http.HandleFunc("/getInfo", handleGetMovieInfo)
+	http.HandleFunc("/getMovies", s.handleGetMovies)
+	http.HandleFunc("/getInfo", s.handleGetMovieInfo)
 	http.HandleFunc("/playFile", handlePlayFile)
 	http.HandleFunc("/stopFile", handleStopFile)
 	http.HandleFunc("/ws", s.handleWebSocket)
 	// Used to test the websocket
 	http.HandleFunc("/", handleSendRoot)
 }
 
-func (Server) Serve() {
+func (Server) Serve(port int) {
 	// Start the server
-	log.Println("Starting server on port 8080")
-	log.Fatal(http.ListenAndServe(":8080", nil))
+	log.Printf("Starting server on port %d\n", port)
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), nil))
```

Comment:
we want to listen on 0.0.0.0: port number.

Reply:
Sounds good. I was keeping with syntax and thought it defaulted to that, but it appears it just defaults to `127.0.0.1`, I'll go make that change as well